ocaml/asmcomp/cmmgen.ml

```diff
@@ -1331,35 +1328,23 @@ and transl_prim_3 env p arg1 arg2 arg3 dbg =
 
   (* Effects *)
   | Presume ->
-      Misc.fatal_error "Effects-related primitives not yet supported"
-      (*
-      (* CR mshinwell: Rc_normal may be wrong, but this code is unlikely
-         to be in production by then *)
+      (* CR mshinwell: check Rc_normal *)
       Cop (Capply (typ_val, Rc_normal),
            [Cconst_symbol ("caml_resume", dbg);
            transl env arg1; transl env arg2; transl env arg3],
            dbg)
-      *)
   | Prunstack ->
-      Misc.fatal_error "Effects-related primitives not yet supported"
-      (*
-      (* CR mshinwell: Rc_normal may be wrong, but this code is unlikely
-         to be in production by then *)
+      (* CR mshinwell: check Rc_normal *)
```

Comment:
`Rc_normal` is fine here but unlikely to ever be a tail call (I think usages of this primitive are never generated in tail position). I'd probably change to `Rc_nontail` for clarity.